TranslatorReasonerAPI.yaml

```
@@ -381,12 +381,12 @@ components:
           example: upregulates
```

```suggestion
          $ref: '#/components/schemas/CURIE'
          example: RO:0002213
```
The `relation` should also be constrained to a CURIE, correct? Note that the suggested changes above should also remove `type: string` on line 380 however GitHub won't allow me to comment on that line directly.

---

I don't know if `relation` is meant to be a CURIE; we should just follow the lead of the data modeling committee here. Is there a place that those sorts of decisions are recorded? Regardless, that can be a separate PR.